study-builder/studies/testboston/emails/account_created.en.html

```diff
@@ -0,0 +1,14 @@
+#set( $text = {
```

Comment:
Idea here is that we define all the `text` strings here in the desired language, then do a include of the template file. The `#parse` directive below does the include and will do substitutions. The template will make use of these `text` strings.